twins/twins.go

```diff
@@ -71,8 +71,26 @@ type TwinRepository interface {
 	RetrieveByAttribute(ctx context.Context, channel, subtopic string) ([]string, error)
 
 	// RetrieveAll retrieves the subset of twins owned by the specified user.
-	RetrieveAll(context.Context, string, uint64, uint64, string, Metadata) (Page, error)
+	RetrieveAll(ctx context.Context, owner string, offset, limit uint64, name string, metadata Metadata) (Page, error)
 
 	// Remove removes the twin having the provided identifier.
 	Remove(ctx context.Context, id string) error
```

Comment:
Definition is `func (tcm twinCacheMiddleware) Remove(ctx context.Context, twinID string)`. You could use `twinID` here